south-african-journal-of-geology.csl

```diff
@@ -0,0 +1,165 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>South African Journal of Geology</title>
+    <title-short>Geol. South Africa</title-short>
+    <id>http://www.zotero.org/styles/south-african-journal-of-geology</id>
+    <link rel="self" href="http://www.zotero.org/styles/south-african-journal-of-geology"/>
```

Comment:
Can you add a "template" link? See https://github.com/citation-style-language/styles/wiki/Style-Requirements#8---template-link